nilearn/signal.py

```diff
@@ -148,6 +149,17 @@ def _detrend(signals, inplace=False, type="linear", n_batches=10):
     return signals
 
 
+def _check_wn(btype, freq, nyq):
+    wn = freq / nyq
```

Comment:
I think that you should cast nyq to a float: float(nyq), just in case some integer finds its way here.